app/controllers/YourAccountController.scala

```diff
@@ -106,6 +98,28 @@ class YourAccountController @Inject()(
           logger.error(s"there was an error getting periods with outstanding amounts $error")
           throw new Exception(error.toString)
       }
+  }
+
+  private def getFilteredPeriodsWithOutstandingAmounts(vatReturnsWithFinancialData: Seq[VatReturnWithFinancialData]) = {
+    financialDataService
+      .filterIfPaymentIsOutstanding(vatReturnsWithFinancialData)
+      .map(vatReturnWithFinancialData =>
+        vatReturnWithFinancialData.vatOwed match {
```

Comment:
I wanted to check this part but never got time. In a scenario when we get back 'vatOwed' from vatReturnWithFinancialData, we just use that by default and don't add in the correction amount. Can we just double confirm that the vatOwed will have the correction amount included in it by default

Reply:
To Discuss with ETMP